.github/actions/install-parcels/action.yml

```
@@ -0,0 +1,39 @@
+name: Setup Anaconda and install parcels
+description: > 
+  In-repo composite action to setup Anaconda and install parcels. Installation of parcels relies on
+  `setup.py` file being available in the root. For general setup of Anaconda environments, just use
+  the `conda-incubator/setup-miniconda` action (setting C variables as required)
+inputs:
+  environment-file:
+    description: "Anaconda environment file"
+    required: true
+  environment-name:
+    description: "Anaconda environment name"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@v2.2.0
+      with:
+        miniconda-version: "latest"
+        activate-environment: ${{ inputs.environment-name }} # Needs to be the same as listed in the env file
+        environment-file: ${{ inputs.environment-file }}
+        auto-activate-base: false
+        auto-update-conda: true
+    - run: conda info
+      shell: bash -el {0}
+    - name: Install parcels
+      run: python setup.py install
+      shell: bash -el {0}
+    - name: Set env variables for macos
+      run: |
+        if [[ "${{ runner.os }}" == mac* ]]; then
```

```suggestion
        if [[ "${{ runner.os }}" == macOS ]]; then
```
No need for wildcard match as values of `runner.os` are `macOS`, `Linux`, or `Windows` as per the [docs](https://docs.github.com/en/actions/learn-github-actions/variables#:~:text=example%2C%20Hosted%20Agent-,RUNNER_OS,-The%20operating%20system)